packages/block-library/src/form-input/deprecated.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import removeAccents from 'remove-accents';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	RichText,
+	__experimentalGetBorderClassesAndStyles as getBorderClassesAndStyles,
+	__experimentalGetColorClassesAndStyles as getColorClassesAndStyles,
+} from '@wordpress/block-editor';
+
+const getNameFromLabelV1 = ( content ) => {
+	const dummyElement = document.createElement( 'div' );
+	dummyElement.innerHTML = content;
+	// Get the slug.
+	return (
+		removeAccents( dummyElement.innerText )
+			// Convert anything that's not a letter or number to a hyphen.
+			.replace( /[^\p{L}\p{N}]+/gu, '-' )
+			// Convert to lowercase
+			.toLowerCase()
+			// Remove any remaining leading or trailing hyphens.
+			.replace( /(^-+)|(-+$)/g, '' )
+	);
+};
```

Comment:
This is copied from the `getNameFromLabel()` function present in `save.js` and used for version 1 deprecation. This is because I think deprecations may not be handled correctly if the original function is updated and the output markup changes.